okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -1025,6 +1034,21 @@ open class OkHttpClient internal constructor(
       pingInterval(duration.toMillis(), MILLISECONDS)
     }
 
+    /**
+     * Sets minimum outbound web socket message size (in bytes) that will be compressed.
+     *
+     * Set to 0 to enable compression for all outbound messages.
+     *
+     * 1024 by default.
+     */
+    fun webSocketMinimumDeflateSize(bytes: Long) = apply {
```

Comment:
Rename to `webSocketMinCompressSize`

“Compress” because developers don’t know what deflate is; that algorithm is an implementation detail.
“Min” for consistency with ConnectionPool and Dispatcher which use the short form for max.

Also tempted to rearrange the words, putting 'min' first...

`minWebSocketMessageToCompress`
`minCompressMessageSize`
`minMessageToCompress`
`minMessageSizeToCompress`

Any favorites?

Reply:
Agree.
I'd go with `minWebSocketMessageToCompress`.